eq-author/src/App/QuestionnaireDesignPage/index.js

```diff
@@ -193,8 +193,9 @@ export class UnwrappedQuestionnaireDesignPage extends Component {
       throw new Error(ERR_PAGE_NOT_FOUND);
     }
 
+    const title = this.getTitle();
     return (
-      <BaseLayout questionnaire={questionnaire}>
+      <BaseLayout questionnaire={questionnaire} title={loading ? "" : title}>
```

Comment:
I think this can become `title={this.getTitle("")}` as it is already handling the loading state there

Reply:
Problem here is, while the page is loading, the title does not exist, giving me a proptype error when opening a questionnaire, because it is not received a title.

Reply:
Changed to get the questionnaire displayName from questionnaire json directly.